Libplanet/Tx/UnexpectedlyTerminatedTxRehearsalException.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Libplanet.Action;
+
+namespace Libplanet.Tx
+{
+    /// <summary>
+    /// The exception that is thrown during a <see cref="Transaction{T}"/>
+    /// is being created when one of <see cref="Transaction{T}.Actions"/>
+    /// throws some exception during rehearsal mode.
+    /// <para>The actual exception that the <see cref="Action"/> threw
+    /// is stored in the <see cref="Exception.InnerException"/> property.</para>
+    /// </summary>
+    public class UnexpectedlyTerminatedTxRehearsalException : Exception
```

Comment:
- Could we seal this class?
- It would be better if `[Serializable]` was attached to make it serializable.

Reply:
Both ideas are good.  I'll do that!

Reply:
@longfin I've just amended the last commit.